linalg/vector.hpp

```diff
@@ -141,11 +141,11 @@ class Vector
    inline void NewMemoryAndSize(const Memory<double> &mem, int s, bool own_mem);
 
    /// Reset the Vector to be a reference to a sub-vector of @a base.
-   inline void MakeRef(Vector &base, int offset, int size);
+   inline void MakeRef(const Vector &base, int offset, int size);
```

Comment:
I don't think adding `const` here is the correct way to create sub-vectors of a `const Vector`. We do not want the interface to remove `const`-ness implicitly. This should be done explicitly in user code with `const_cast<Vector &>(...)`.

Reply:
Okay, we can make that change in our application code.

Reply:
Just reverted the change.